start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,725 @@
+function NewEvaluationPage(options){
+    
+    const elements = [
+        {level:'Trainee'},{level:'Junior'},{level:'Middle'}, {level:'Senior'}
+    ];
+
+    const selector = [
+        {
+            value:'1',
+            language: 'Evaluation',
+        },
+        {
+            value:'2',
+            language: 'javascript',
+        },
+        {
+            value:'3',
+            language: 'Java',
+        },
+        {
+            value:'4',
+            language: 'Haskell',
+        }
+    ];
+
+    const description = [
+        {
+            selectDescription: 'Evaluation'
+        },
+        {
+            selectDescription: 'Exception handling'
+        },
+        {
+            selectDescription: 'Version Control'
+        },
+        {
+            selectDescription: 'Access modifiers'
+        },
+        {
+            selectDescription: 'Design Pattern'
+        },
+        {
+            selectDescription: 'Design Pattern'
+        },
+        {
+            selectDescription: 'Polymorphism'
+        },
+        {
+            selectDescription: 'RegEx'
+        }
+
+    ];
+
+
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    ${NewEvaluationTable({
+        el: elements,
+        sel: selector,
+        des: description
+    })}
+    ${FooterNew()}
+
+
+    `;
+}
+
+const TechnicalLevel = function(options={}){
+    const technical = [];
+    options.el.forEach(function(n){
+        technical.push(`<td class='technical-level-td'>${n.level}</td>`);
+    });
+    return technical.join('');
+    
+}
+
+const SelectLevel = function(options={}){
+    const selectArr = [];
+    options.sel.forEach(function(n){
+        selectArr.push(`
+            <option value="${n.value}">${n.language}</option>
+            `);
+    });
+    return selectArr.join('');
+}
+
+// const SelectName = function(options={}){
```

Comment:
Would this piece of code be used in the near future? If not, might as well remove it and get into the habit of removing unused pieces of code in general as it adds clutter to the project as it grows

Reply:
Solved